src/components/LoginForm.js

```diff
@@ -0,0 +1,16 @@
+import BaseComponent from "./BaseComponent";
+import RegistartionForm from "./RegistartionForm";
+
+export default class LoginForm extends BaseComponent{
+    constructor(page) {
+        super(page, 'modal-content');
+        this.registartionForm = new RegistartionForm(this._page);
+        this._registartionBtn = page.locator('.btn-link').last();
```

Comment:
For components, we need to have a container locator as a base.
In your case - this._registartionBtn= this._compomemt.locator('.btn-link').last();